dbaccess/transaction_output.go

```diff
@@ -47,14 +47,13 @@ func TransactionOutputsByOutpoints(ctx Context, outpoints []*Outpoint) ([]*dbmod
 
 	var dbPreviousTransactionsOutputs []*dbmodels.TransactionOutput
 	// fetch previous transaction outputs in chunks to prevent too-large SQL queries
-	for offset := 0; offset < len(outpointTuples); {
+	for i, offset := 0, 0; offset < len(outpointTuples); i++ {
 		var chunk [][]interface{}
 		chunk, offset = outpointsChunk(outpointTuples, offset)
 		var dbPreviousTransactionsOutputsChunk []*dbmodels.TransactionOutput
-
 		dbResult := db.
-			Joins("LEFT JOIN `transactions` ON `transactions`.`id` = `transaction_outputs`.`transaction_id`").
-			Where("(`transactions`.`transaction_id`, `transaction_outputs`.`index`) IN (?)", chunk).
+			Joins("LEFT JOIN transactions ON transactions.id = transaction_outputs.transaction_id").
+			Where("transactions.transaction_id in (?) AND transaction_outputs.index IN (?)", chunk[i][0], chunk[i][1]).
```

Comment:
Wait, do you want to tell me postgres doesn't support `IN ((a,b),(c,d))` type of queries?

If so:
1. The whole chunks things is redundant.
2. The whole thing is going to be really slow.
3. Maybe a new approach is desirable.

Reply:
My bad. The original query works perfectly fine in postgres